robot_log_visualizer/file_reader/signal_provider.py

```diff
@@ -11,6 +11,10 @@
 import idyntree.swig as idyn
 
 
+# for real-time logging
+import yarp
```

Comment:
@GiulioRomualdi should we lazy load also yarp or we just need to lazy load blf?

fyi @nicktrem